### DISCUSSION TITLE
Installed in on Windows with VS022 - doesn't do anything - What am I doing wrong?

### DISCUSSION THREAD

**Discussion by alainlaflamme** [opensourcecobol/opensourcecobol4j#491] [Q&A]
I did all the steps to run it with Windows.  When I run cobj.exe, it just flashes and closes directly.
What am I missing?

**Comment by kio-watanabe** [comment]
cobj.exe is not intended to be invoked directly.
If you have completed the preparations to run cobj, you can run `cobj` command from the command line.

Open powershell or command prompt and execute `cobj` command against the COBOL file.
Then, a java file and class files should be generated.
If the java file can be successfully executed, the setup is complete.

Example:
HELLOWORLD.cbl
 ```
      IDENTIFICATION              DIVISION.
       PROGRAM-ID.                 HELLOWORLD.
       DATA                        DIVISION.
       WORKING-STORAGE             SECTION.
       PROCEDURE                   DIVISION.
           DISPLAY "HELLO WORLD!".
           STOP RUN.
```
Screen at runtime
![image](https://github.com/user-attachments/assets/c78f1c68-6acc-43b8-83e7-289d706d09d2)

If you get an error, read the README and check the environment variable settings and the placement of cobj.exe, libcobj.jar, and config files.

If the problem is still not resolved, please contact us again.
